simple/stats/stat_var_hierarchy_constants.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Set of properties that should not be considered for building the hierarchy.
```

Comment:
Please include links to where you got it from so we keep it in sync when that changes?

We have 2 copies now:  g3, dc-import.

Reply:
Done. Please see comment about subset of properties. Should we just include them all?

Reply:
In the interest of keeping code in sync with something in g3, keep them?

Reply:
Done.